helpers/englishUlbHelpers.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * ugntHelpers.js - this is the code called by ugntParse.js to download and convert greek resources (was BHP and now
+ *                  UGNT.
+ */
+
+//TODO: EN_ULB_URL path is a temporary repo, needs to be updated to final location.
+
+import path from 'path-extra';
+import * as bible from '../scripts/bible';
+import * as UsfmParseHelpers from './usfmParseHelpers';
+
+const EN_ULB_URL = 'https://git.door43.org/photonomad0/AlignedUlb_en/raw/master';
```

Comment:
I thought this was supposed to come from the catalog?

Reply:
Long term yes, but they still have to align it all.  This is temporary, that's why I added the TODO above.